src/tiles/expandable-tile.component.ts

```diff
@@ -39,6 +43,12 @@ export interface ExpandableTileTranslations {
 	`
 })
 export class ExpandableTile implements AfterContentInit {
+	@HostBinding("class.bx--tile--light") get lightThemeEnabled() {
+		return this.theme === "light";
+	}
```

Comment:
Same thing as the clickable-tile here. Why do we need to assign the `bx--tile--light` class to the host & the button?